Diana's Files/Advising.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<title>W3.CSS Template</title>
```

Comment:
Maybe we could use BAResources as the title for this site. 
Something similar in the Events page. 
Other than that, LGTM.